aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2614,7 +2637,7 @@ std::vector<Tensor> dsplit(const Tensor& self, int64_t split_size) {
   return at::tensor_split(self, split_size, 2);
 }
 
-std::vector<Tensor> split_with_sizes(const Tensor& self, IntArrayRef split_sizes, int64_t dim) {
+std::vector<Tensor> split_with_sizes(const Tensor& self, IntArrayRef split_sizes, int64_t dim, bool drop_remainder) {
```

Comment:
This may not be related to this commit. In line 2652, I think they mean 
```
 "entries, but got split_size=", length);
```
instead of 
```
             "entries, but got split_sizes=", split_sizes);
```

Reply:
Sorry, omit my comment. I think it is correct.